net/snort3/files/snort.init

```diff
@@ -38,17 +38,30 @@ start_service() {
 	if [ "$manual" = 0 ]; then
 		local config_file=$($MGR setup)
 		procd_set_param command "$PROG" -q -c "${config_file}"
+		jail
 	else
 		procd_set_param command $PROG -q -i "$interface" -c "${config_dir%/}/snort.lua" --tweaks local
 		procd_set_param env SNORT_LUA_PATH="$config_dir"
 		procd_set_param file $CONFIGFILE
+		jail
 	fi
```

Comment:
If all of the branches run `jail` at the end, why not put it outside of the `if` statement?

Reply:
good point ... that actually eliminates the need for a function all together